### PR TITLE
Upload of Individual Assignment2a

### PR DESCRIPTION
**Pull request by MarcoAS99**
Upload of Individual Assignment2a

### CODE REVIEW

File: Assignment2/Marco_turtle.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@prefix cl: <http://www.oeg-upm.net/Class01#/>.
```
Comment (ocorcho) [inline Assignment2/Marco_turtle.ttl:1]: No /# should be used

File: Assignment2/Marco_turtle.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@prefix s: <http://www.oeg-upm.net/Sensor029#/>.
+@prefix pc: <http://www.oeg-upm.net/Computer101#/>.
+@prefix m: <http://www.oeg-upm.net/Measurement8401#/>.
```
Comment (ocorcho) [inline Assignment2/Marco_turtle.ttl:4]: Needed?

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by MarcoAS99** [comment]
Assignment reviewed.

File: Assignment2/MarcoAS99-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                },
+                {
+                    "@id": "rec:Sensor029",
+                    "@type": "rec"
+                }
+            ]
+        },
+        {
+            "@id": "rec:Measurement8401",
+            "@type": "rec",
```
Comment (ocorcho) [inline Assignment2/MarcoAS99-fromRDFtoJSON-LD.jsonld:30]: You are using types wrongly

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Assignment 2a ok, request for changes in 2b

**Pull request closed without merging**